### PR TITLE
docs: update quickstart

### PR DESCRIPTION
**Pull request by ZeroMagic**
Signed-off-by: ZeroMagic <anthonyliu@zju.edu.cn>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,14 +39,19 @@ cd $GOPATH/src/github.com/ZJU-SEL/capstan
 make && make install
 ```
 
-Install Prometheus and Grafana:
+Prepare Kubernetes admin config file:
```
Comment (mozhuli) [inline README.md]: Add steps of install kubectl

File: deploy/quickstart.sh [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    kubectl delete ns exporter
+    printf 'deleting exporter namespace'
+    while [ -n "$(kubectl get ns | grep exporter)" ]
+    do
+        printf '.'
+    done
+    printf '\n'
+fi
+sleep 5
+kubectl create namespace exporter
```
Comment (mozhuli) [inline deploy/quickstart.sh]: use capstan-exporter is better

File: deploy/quickstart.sh [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+kubectl create -f $GOPATH/src/github.com/ZJU-SEL/capstan/deploy/node-exporter.yaml
+
+# Get user's HostIP
+IPADDR=$(curl -s http://members.3322.org/dyndns/getip)
+sed -i "s/<Your-HostIP>/${IPADDR}/g" $GOPATH/src/github.com/ZJU-SEL/capstan/deploy/grafana-datasources.yaml $GOPATH/src/github.com/ZJU-SEL/capstan/examples/capstan.conf
+
+# Run Prometheus
+mkdir -p /etc/capstan/prometheus
```
Comment (mozhuli) [inline deploy/quickstart.sh]: put in Makefile install section is better

**Review by mozhuli — CHANGES_REQUESTED** [review]

**Comment by ZeroMagic** [comment]
@mozhuli I have changed the code.

**Review by mozhuli — APPROVED** [review]

**Pull request merged**